src/test/java/seedu/medibook/logic/commands/CommandTestUtil.java

```diff
@@ -99,8 +99,10 @@
     public static final String INVALID_BLOOD_TYPE_DESC = " " + PREFIX_BLOOD_TYPE + "C+"; // 'C+' invalid blood type
     public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "hubby*"; // '*' not allowed in tags
     public static final String INVALID_NOTE_DATE_DESC = " " + PREFIX_DATE + "2-3-2020"; // incorrect date format
-    public static final String INVALID_DOCTOR_NAME_DESC = " " + PREFIX_NAME + "A* doctor"; // '* not allowed in name'
-    public static final String INVALID_DOCTOR_MCR_DESC = " " + PREFIX_MCR + "Z82730P"; // incorrect mcr format
+    // '* not allowed in name'
```

Comment:
Should this be "'*' not allowed in name" instead of having quotations around the whole comment?

Reply:
Thanks for pointing it out. I have fixed it.